File: WaaS/WaaS.Shared/Entities/ScrapeJob.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace WaaS.Shared.Entities
{
  public class ScrapeJob
  {
    public int Id { get; set; }
    public bool Enabled { get; set; }
    public string Url { get; set; }
    public string Pattern { get; set; }
    public string AlternativeEmail { get; set; }
  }
}

File: WaaS/WaaS.Presentation/ClientApp/src/app/overview/scrape-job.ts
import { ScrapeJobStatusCode } from './scrape-job-status/scrape-job-status-code';

export class ScrapeJob {
  id: number;
  name: string;
  url: string;
  pattern: string;
  enabled: boolean;
  alternativeEmail: string;
  status: ScrapeJobStatusCode;
}
